vochain/account.go

```diff
@@ -89,13 +93,10 @@ func (a *Account) DelDelegate(addr common.Address) error {
 // and updates the state with the new values (including nonce).
 // If origin address acc is not enough, ErrNotEnoughBalance is returned.
 // If provided nonce does not match origin address nonce+1, ErrAccountNonceInvalid is returned.
-// If isQuery is set to true, this method will only check against the current state (no changes will be stored)
-func (v *State) TransferBalance(from, to common.Address, amount uint64, nonce uint32, isQuery bool) error {
+func (v *State) TransferBalance(from, to common.Address, amount uint64, nonce uint64, faucet bool) error {
```

Comment:
Same as above. I don't like using the same Method for collect faucet funds and send tokens between accounts. I think in this case its better to make things very explicit.